HLTrigger/Configuration/python/Phase2/ESProducersSources/XMLIdealGeometryESSource_cfi.py

```diff
@@ -0,0 +1,123 @@
+import FWCore.ParameterSet.Config as cms
+
+XMLIdealGeometryESSource = cms.ESSource("XMLIdealGeometryESSource",
```

Comment:
if/when the HLT setup can be supported by a cmsDriver/ConfigBuilder, I suppose that this file will not be needed and a relevant ```Geometry/CMSCommonData/python/cmsExtendedGeometry2023D*``` will be loaded instead.

Reply:
I agree, but I would like to factorize getting this into the release vs making it supported by cmsDriver.

Reply:
I see, is the plan to make someCfg.py, which can then be executed with cmsRun directly?

Reply:
For the time being, yes. But I am open to suggestions that we should do it on a different way.